Xamarin.Forms.Platform.WPF/WPFPlatformServices.cs

```diff
@@ -151,6 +151,18 @@ public void StartTimer(TimeSpan interval, Func<bool> callback)
 			};
 		}
 
+		public void StartTimer(TimeSpan interval, Func<Task<bool>> callback)
+		{
+			var timer = new DispatcherTimer(DispatcherPriority.Background, System.Windows.Application.Current.Dispatcher) { Interval = interval };
+			timer.Start();
+			timer.Tick += async (sender, args) =>
```

Comment:
sender and args are not used

Reply:
`DispatcherTimer.Tick` is an EventHandler which can only subscribe to the following method signature: `void (object sender, EventArgs e)`